django_ipam/management/commands/__init__.py

```diff
@@ -0,0 +1,45 @@
+import csv
+from ipaddress import ip_network
+
+from django.core.management.base import BaseCommand, CommandError
+
+
+class BaseImportSubnetCommand(BaseCommand):
+    help = 'Import an entire Subnet data from a csv/xls/xlsx file'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--file', type=str, help="File path to import csv data.")
+
+    def handle(self, *args, **options):
+        file = options['file']
+        try:
+            if not file.endswith(('.csv', '.xls', '.xlsx')):
+                raise CommandError('File type not supported')
+            with open(file, 'rb+') as csvfile:
+                if not self.subnet_model.import_csv(self, csvfile):
+                    raise CommandError('Import data error.')
+            csvfile.close()
+        except FileNotFoundError:
+            raise CommandError('File "%s" not found.' % file)
+        self.stdout.write(self.style.SUCCESS('Successfully imported "%s"' % file))
+
+
+class BaseExportSubnetCommand(BaseCommand):
+    help = 'Export an entire Subnet data as a csv file'
+
+    def add_arguments(self, parser):
+        parser.add_argument('subnet', type=str, help='Subnet to export.')
+
+    def handle(self, *args, **options):
+        subnet = options['subnet']
+        try:
+            instance = self.subnet_model.objects.get(subnet=ip_network(subnet))
+        except self.subnet_model.DoesNotExist:
+            raise CommandError('Subnet "%s" does not exist' % subnet)
+        except ValueError:
+            raise CommandError("'%s' does not appear to be an IPv4 or IPv6 network" % subnet)
```

Comment:
i18n issue, find out how to use string interpolation in translatable strings here (look for "Python’s standard named-string interpolation syntax"):
https://docs.djangoproject.com/en/dev/topics/i18n/translation/#standard-translation